core/flyout_base.js

```diff
@@ -596,6 +596,33 @@ Blockly.Flyout.prototype.onMouseDown_ = function(e) {
   }
 };
 
+/**
+ * Helper function to get the list of variables that have been added to the
+ * workspace after adding a new block, using the given list of variables that
+ * were in the workspace before the new block was added.
+ * @param {!Array.<Blockly.VariableModel>} originalVariables The array of
```

Comment:
!Array.<!Blockly.VariableModel>, here and on the return.  And then we're done, and I'll merge it as soon as travis finishes!